source/NVDAObjects/window/edit.py

```diff
@@ -581,6 +567,32 @@ def _getFormatFieldAtRange(self, textRange, formatConfig):
 			pass
 		return formatField
 
+	def _getFormatFieldColor(self, fontObj, formatField):

```

Comment:
same here

Reply:
Fixed in bc4887832cf903beee219772e506fc8cd9a0f5b1. I have left fontObj without type hint.